File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// BluefinTecsUserBackoffice SDK exists test

require_once __DIR__ . '/../bluefintecsuserbackoffice_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = BluefinTecsUserBackofficeSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: kotlin/test/ExistsTest.kt
package voxgig.bluefintecsuserbackofficesdk.sdktest

import org.junit.jupiter.api.Assertions.assertNotNull
import org.junit.jupiter.api.Test

import voxgig.bluefintecsuserbackofficesdk.core.BluefinTecsUserBackofficeSDK

class ExistsTest {

  @Test
  fun testMode() {
    val testsdk = BluefinTecsUserBackofficeSDK.testSDK()
    assertNotNull(testsdk, "expected non-nil SDK")
  }
}
